public/apps/configuration/app-router.tsx

```diff
@@ -147,6 +150,15 @@ export function AppRouter(props: AppDependencies) {
         </Route>
         <EuiPageBody>
           <Switch>
+            <Route
+              path={buildUrl(ResourceType.roles, Action.view, ':roleName', SubAction.mapuser)}
```

Comment:
I think view is used for read only page. This looks like Action.edit

Reply:
It depends on whether we want to consider 'mapuser' as general action. According to me mapuser is not a general action. It is specific to role. But yes let's wait for other reviewer's opinion on this.

Reply:
Agreed on both. So in the url can we use role/edit/roleName/mapuser?

Reply:
Changed the url to `role/edit/roleName/mapuser`

![image](https://user-images.githubusercontent.com/63824432/88978278-3ffff600-d274-11ea-98a4-c71658406528.png)